### PR TITLE
Reference naming convention

### PR DESCRIPTION
**Pull request by anders-swanson**

### CODE REVIEW

**Review by rosemarymarano — APPROVED** [review]
Much better!!!

File: content/en/docs/reference/API/Verrazzano/v1beta1.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Verrazzano V1Beta1 Custom Resource Definition
+title: Verrazzano (v1beta1)
```
Comment (gilbode) [inline content/en/docs/reference/API/Verrazzano/v1beta1.md:2]: Kubernetes doesn't include the parenthesis when they have versioned resources - https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2beta2/ - wonder if that might look a little better.

File: content/en/docs/reference/API/Verrazzano/v1beta1.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 linkTitle: Verrazzano CRD (v1beta1)
```
Comment (gilbode) [inline content/en/docs/reference/API/Verrazzano/v1beta1.md:3]: This is still going to say CRD in the title - don't think we should have CRD.

File: content/en/docs/reference/API/Verrazzano/v1beta1.md (hunk 1 above)
Comment (anders-swanson) [reply content/en/docs/reference/API/Verrazzano/v1beta1.md]: it does. updated.

File: content/en/docs/reference/API/Verrazzano/v1beta1.md (hunk 2 above)
Comment (anders-swanson) [reply content/en/docs/reference/API/Verrazzano/v1beta1.md]: removed CRD for all

**Pull request merged**